src/webapi/xml_http_request.rs

```diff
@@ -0,0 +1,129 @@
+use webapi::event_target::{IEventTarget, EventTarget};
+use webcore::value::{
+    Reference,
+    Value,
+};
+use webcore::try_from::TryInto;
+
+/// XMLHttpRequest object to make http requests
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest)
```

Comment:
Please copy the first paragraph from MDN, and also put an extra newline before `[(JavaScript docs)]`